_typos.toml

```diff
@@ -11,6 +11,7 @@ extend-ignore-re = [
     "FrIeNdLy-\\._\\.-bArD",
     "I borken you cache",
     "eb1ba5f5",
+    "github_pat_.*"
```

Comment:
Had to add this otherwise the pre-commit hook would think `Nd` should be `And` in `pip_install.rs`